templates/Bulk_RNA_seq/README.md

```diff
@@ -0,0 +1,8 @@
+# Bulk RNA-seq analysis pipeline
+
+This is a Bulk RNA-seq analysis pipeline template in the catalog. 
+
+- `{name}/{name}.ipynb`: The template iteself
+- `{name}/README.md`: Long description of template
+- `{name}/requirements.txt`: Python dependencies necessary to run your template
+- `{name}/template.json`: Metadata about your template to be used for the website
```

Comment:
This should be replaced with an actual README pertinent to your template. This will become a part of the template intro webpage, this `{name}/{name}` stuff is irrelevant.